### ISSUE TITLE
deepstream 使用mmdetection-to-tensorrt转换的Mask Rcnn模型 无法检出mask,请问该如何配置.

### ISSUE THREAD

**Issue by pangchao-git** [grimoire/mmdetection-to-tensorrt#115]
1 模型转化时候这样修改了Infrence.py
trt_model = mmdet2trt(cfg_path, args.checkpoint, output_names=["num_detections", "boxes", "scores", "classes","masks"], fp16_mode=args.fp16, device=args.device,enable_mask=True)
    torch.save(trt_model.state_dict(), args.save_path)
2 deepstream 的配置文件

output-bbox-name=bbox
output-blob-names=num_detections;boxes;scores;classes;masks
parse-bbox-func-name=NvDsInferParseMmdet

custom-lib-path=nvdsinfer_custom_impl_fasterRCNN/libamirstan_plugin.so
